PR#2385 – Liquid Exception

- [ ] Liquid Exception: Could not find document '' in tag 'link'. Make sure the document exists and the path is correct. in /home/runner/work/blog/blog/_posts/2025/11/2025-11-19-SCAGWilsonSupportsKabulStyleRaidsChicago.md

```
Run bundle exec jekyll build --trace --incremental --baseurl "/blog"
  bundle exec jekyll build --trace --incremental --baseurl "/blog"
  shell: /usr/bin/bash -e {0}
  env:
    GITHUB_PAGES: true
    JEKYLL_ENV: production
    JEKYLL_GITHUB_TOKEN: 
    LOG_LEVEL: debug
To use retry middleware with Faraday v2.0+, install `faraday-retry` gem
Configuration file: /home/runner/work/blog/blog/_config.yml
            Source: /home/runner/work/blog/blog
       Destination: /home/runner/work/blog/blog/_site
 Incremental build: enabled
      Generating... 
  AI Related Posts: Creating cache [.ai_related_posts_cache.sqlite3]
  AI Related Posts: Fetch disabled. Using cached related posts data.
  AI Related Posts: Found 0 cached embeddings; skipped 1368 fetches
       Jekyll Feed: Generating feed for posts
         AutoPages: Disabled/Not configured in site.config.
        Pagination: ----------------------------
        Pagination: Page: blogpager.html
        Pagination:  Active configuration
        Pagination:   Enabled:          true
        Pagination:   Items per page:   15
        Pagination:   Permalink:        /page/:num/
        Pagination:   Title:            :title - page :num of :max
        Pagination:   Limit:            0
        Pagination:   Sort by:          date
        Pagination:   Sort reverse:     true
        Pagination:  Active Filters
        Pagination:   Collection:       posts
        Pagination:   Offset:           0
        Pagination:   Category:         [Not set]
        Pagination:   Tag:              [Not set]
        Pagination:   Locale:           [Not set]
        Pagination:  Filtering by: Category  1368 => 1368
        Pagination:  Filtering by: Tag       1368 => 1368
        Pagination:  Filtering by: Locale    1368 => 1368
        Pagination: Rolling through the date fields for all documents
        Pagination: Complete, processed 1 pagination page(s)
  Liquid Exception: Could not find document '' in tag 'link'. Make sure the document exists and the path is correct. in /home/runner/work/blog/blog/_posts/2025/11/2025-11-19-SCAGWilsonSupportsKabulStyleRaidsChicago.md
bundler: failed to load command: jekyll (/home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/bin/jekyll)
/home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/tags/link.rb:32:in 'Jekyll::Tags::Link#render': Could not find document '' in tag 'link'. (ArgumentError)
Make sure the document exists and the path is correct.
        raise ArgumentError, <<~MSG
              ^^^^^^^^^^^^^^^^^^^^^
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/liquid-4.0.4/lib/liquid/block_body.rb:103:in 'Liquid::BlockBody#render_node_to_output'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/liquid-4.0.4/lib/liquid/block_body.rb:91:in 'Liquid::BlockBody#render'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/liquid-4.0.4/lib/liquid/template.rb:206:in 'block in Liquid::Template#render'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/liquid-4.0.4/lib/liquid/template.rb:240:in 'Liquid::Template#with_profiling'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/liquid-4.0.4/lib/liquid/template.rb:205:in 'Liquid::Template#render'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/liquid-4.0.4/lib/liquid/template.rb:218:in 'Liquid::Template#render!'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/liquid_renderer/file.rb:39:in 'block (3 levels) in Jekyll::LiquidRenderer::File#render!'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/liquid_renderer/file.rb:59:in 'Jekyll::LiquidRenderer::File#measure_counts'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/liquid_renderer/file.rb:38:in 'block (2 levels) in Jekyll::LiquidRenderer::File#render!'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/liquid_renderer/file.rb:63:in 'Jekyll::LiquidRenderer::File#measure_bytes'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/liquid_renderer/file.rb:37:in 'block in Jekyll::LiquidRenderer::File#render!'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/liquid_renderer/file.rb:70:in 'Jekyll::LiquidRenderer::File#measure_time'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/liquid_renderer/file.rb:36:in 'Jekyll::LiquidRenderer::File#render!'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/renderer.rb:129:in 'Jekyll::Renderer#render_liquid'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/renderer.rb:80:in 'Jekyll::Renderer#render_document'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/renderer.rb:63:in 'Jekyll::Renderer#run'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:572:in 'Jekyll::Site#render_regenerated'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:557:in 'block (2 levels) in Jekyll::Site#render_docs'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:556:in 'Array#each'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:556:in 'block in Jekyll::Site#render_docs'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:555:in 'Hash#each_value'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:555:in 'Jekyll::Site#render_docs'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:210:in 'Jekyll::Site#render'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:80:in 'Jekyll::Site#process'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/command.rb:28:in 'Jekyll::Command.process_site'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/commands/build.rb:65:in 'Jekyll::Commands::Build.build'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/commands/build.rb:36:in 'Jekyll::Commands::Build.process'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/command.rb:91:in 'block in Jekyll::Command.process_with_graceful_fail'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/command.rb:91:in 'Array#each'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/command.rb:91:in 'Jekyll::Command.process_with_graceful_fail'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/lib/jekyll/commands/build.rb:18:in 'block (2 levels) in Jekyll::Commands::Build.init_with_program'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in 'block in Mercenary::Command#execute'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in 'Array#each'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in 'Mercenary::Command#execute'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/mercenary-0.4.0/lib/mercenary/program.rb:44:in 'Mercenary::Program#go'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/mercenary-0.4.0/lib/mercenary.rb:21:in 'Mercenary.program'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/gems/jekyll-4.4.1/exe/jekyll:15:in '<top (required)>'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/bin/jekyll:25:in 'Kernel#load'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/3.4.0/bin/jekyll:25:in '<top (required)>'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/lib/ruby/3.4.0/bundler/cli/exec.rb:59:in 'Kernel.load'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/lib/ruby/3.4.0/bundler/cli/exec.rb:59:in 'Bundler::CLI::Exec#kernel_load'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/lib/ruby/3.4.0/bundler/cli/exec.rb:23:in 'Bundler::CLI::Exec#run'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/lib/ruby/3.4.0/bundler/cli.rb:452:in 'Bundler::CLI#exec'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/lib/ruby/3.4.0/bundler/vendor/thor/lib/thor/command.rb:28:in 'Bundler::Thor::Command#run'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/lib/ruby/3.4.0/bundler/vendor/thor/lib/thor/invocation.rb:127:in 'Bundler::Thor::Invocation#invoke_command'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/lib/ruby/3.4.0/bundler/vendor/thor/lib/thor.rb:538:in 'Bundler::Thor.dispatch'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/lib/ruby/3.4.0/bundler/cli.rb:35:in 'Bundler::CLI.dispatch'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/lib/ruby/3.4.0/bundler/vendor/thor/lib/thor/base.rb:584:in 'Bundler::Thor::Base::ClassMethods#start'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/lib/ruby/3.4.0/bundler/cli.rb:29:in 'Bundler::CLI.start'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/lib/ruby/gems/3.4.0/gems/bundler-2.6.9/exe/bundle:28:in 'block in <top (required)>'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/lib/ruby/3.4.0/bundler/friendly_errors.rb:117:in 'Bundler.with_friendly_errors'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/lib/ruby/gems/3.4.0/gems/bundler-2.6.9/exe/bundle:20:in '<top (required)>'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/bin/bundle:25:in 'Kernel#load'
	from /opt/hostedtoolcache/Ruby/3.4.7/x64/bin/bundle:25:in '<main>'
Error: Process completed with exit code 1.
```

_Originally posted by @RalphHightower in https://github.com/RalphHightower/blog/issues/2395#issuecomment-3555673929_

--- a/_posts/2025/11/2025-11-19-SCAGWilsonSupportsKabulStyleRaidsChicago.md
+++ b/_posts/2025/11/2025-11-19-SCAGWilsonSupportsKabulStyleRaidsChicago.md
@@ -12,7 +12,7 @@ title: "My Dumb State, South Carolina: Attorney General Alan Wilson Supports Nat
 
 Black Hawk helicopters. Agents rappelling from helicopters. Kicking down doors. Trashing apartments. Apartment residents separated by doing color. Children separated from their parents. 
 
-- 2025-10-16 22:13; [Chicago Is In America, Not Gaza]({% link  %})
+- 2025-10-16 22:13; [Chicago Is In America, Not Gaza]({% link _posts/2025/10/2025-10-16-ChicagoIsInAmericaNotVietnam.md %})
 - 2025-11-02 21:28; [Trump Wants National Guard Strike Force To Quell “Civil Unrest”]({% link _posts/2025/10/2025-10-16-ChicagoIsInAmericaNotVietnam.md %})
 - 2025-11-07 22:52; [Customs and Border Patrol Agent, Charles Exum, Is a Sick, Barbaric Sub-Human. Bragging About Five Rounds Inflicting Seven Bullet Holes]({% link _posts/2025/11/2025-11-07-CharlesExumExcessiveDeadlyForceCBPChicago.md %})
 - 2025-11-14 22:46; [Trump’s ICE Raids In Chicago Raising Alarms]({% link _posts/2025/11/2025-11-14-TrumpChicagoIceRaidsAlarms.md %})

